src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1949,6 +1949,10 @@ void layout_optimizer::select_preferred_formats_for_onednn(program_node& node, d
                 src_fmt = onednn::find_data_format(prim_desc.dst_desc());
             }
 
+            // WA: b_fs_yx_fsv2 has dnnl::memory::format_tag::undef , so select byfx instead of b_fs_yx_fsv2.
```

Comment:
`avoid b_fs_yx_fsv2 because Onednn tag aBcd2b is not declared`

Reply:
Updated the comment. Thanks!